modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutsAdminDisplayContext.java

```diff
@@ -1713,6 +1722,17 @@ public boolean isShowViewLayoutAction(Layout layout) {
 		return false;
 	}
 
+	public boolean isSingleLanguageSite() {
+		Set<Locale> availableLocales = LanguageUtil.getAvailableLocales(
+			themeDisplay.getSiteGroupId());
+
+		if (availableLocales.size() == 1) {
+			return true;
+		}
+
+		return false;
+	}
+
```

Comment:
It seems this isn't use outside of the class. Can we make it private?

Reply:
It's used on 
`LayoutActionDropdownItemsProvider#469` and` LayoutActionDropdownItemsProvider#486`